OCC.Service.Webhost/Repositories/SessionRepository.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using OCC.Data;
+using OCC.Service.Webhost.Services;
+using OCC.Service.Webhost.Tools;
+using Session = OCC.Service.Webhost.Services.Session;
+
+namespace OCC.Service.Webhost.Repositories
+{
+    public class SessionRepository
+    {
+        private readonly OCCDB _dbContext;
+
+        public SessionRepository(OCCDB dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public Session GetSession(int id)
+        {
+            var s = (from x in _dbContext.Sessions.Include("Speaker").Include("Track").Include("Timeslot")
```

Comment:
I also do not like magic strings, but I believe there is a better option for this scenario. You can use a lambda here in the includes instead of the strings. This was pulled over wholesale from master and I still want to add a few tests before changing things around.